webxrlayers-1.bs

```diff
@@ -859,8 +862,8 @@ interface XRWebGLBinding {
                                       optional XREquirectLayerInit init);
   XRCubeLayer createCubeLayer(optional XRCubeLayerInit init);
 
-  XRWebGLSubImage getSubImage(XRCompositionLayer layer, XRFrame frame);
-  XRWebGLSubImage getViewSubImage(XRCompositionLayer layer, XRView view);
+  XRWebGLSubImage getSubImage(XRCompositionLayer layer, XRFrame frame, optional XREye eye = "none");
+  XRWebGLSubImage getViewSubImage(XRProjectionLayer layer, XRView view);
```

Comment:
I really like the idea of explicitly limiting `getViewSubImage()` to `XRProjectionLayer`s! The only question it leaves me with is what happens if I pass a projection layer to `getSubImage()`? It's still an `XRCompositorLayer`, so the IDL won't reject it, but I don't think it's intended to work. We probably need a check in the algorithm below to reject it?

Reply:
I agree that it makes little sense that projection layers are allowed here.
I can't think of a good reason why an author would do that apart.

I will add the rejection of projection layers to the algorithm.